Source/Core/CircleGeometry.js

```diff
@@ -175,5 +175,12 @@ define([
         });
     };
 
+    /**
+     * @private
```

Comment:
Well, the problem I pointed out still exists (can't create a good surface geometry without using a private function), but since it's not a new problem; I'm okay with this.